core/lib/spree/event/adapters/event_bus.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'spree/event/event'
+require 'spree/event/listener'
+
+module Spree
+  module Event
+    module Adapters
+      # Adapter for {Spree::Event}
+      #
+      # Please, access it through {Spree::Event} module. You only need to
+      # configure an instance of it to be used as the default adapter.  E.g., in
+      # `spree.rb` initializer:
+      #
+      # @example
+      #   require "spree/event/adapters/event_bus"
+      #
+      #   Spree.config do |config|
+      #     # ...
+      #     config.events.adapter = EventBus.new
+      #     # ...
+      #   end
+      #
+      # You won't need to do that from Solidus version 4.0 as this adapter will
+      # be the default one.
+      class EventBus
```

Comment:
Should this be simply called `Default`? The actual event bus is `Spree::Event`, this is just an adapter.

Reply:
Ok, that makes sense. The adapter itself is an `EventBus`, as it could be used standalone, but probably it's easier to understand doing it as you say.